test/grid.js

```diff
@@ -1,15 +1,116 @@
-import { setup } from './_helpers.js'
-import { assert, test } from 'vitest'
-
-setup()
-
-test('grid columns', async (t) => {
-  const staticNumbers = ['grid-cols-2', 'grid-cols-5']
-  const arbitrary = ['grid-cols-[200px_minmax(900px,_1fr)_100px]', 'grid-cols-[320px_1fr]']
-  const { css } = await t.uno.generate([staticNumbers, arbitrary].flat(Infinity))
-  const staticResult = n => `grid-template-columns:repeat(${n},minmax(0,1fr));`
-  assert.include(css, staticResult(2))
-  assert.include(css, staticResult(5))
-  assert.include(css, 'grid-template-columns:200px minmax(900px, 1fr) 100px;')
-  assert.include(css, 'grid-template-columns:320px 1fr;')
-})
+import { setup } from "./_helpers.js";
+import { expect, assert, test } from "vitest";
+
+setup();
+
+test("grid span", async (t) => {
+  const classes = [
+    "col-auto",
+    "row-auto",
+    "col-span-full",
+    "row-span-full",
+    "row-span-5",
+    "col-span-3",
```

Comment:
Maybe we should test all values that are allowed within the bounds? 🤔 this would go for all classes with arbitrary values.

Alongside that we should probably also test that anything outside of the bounds doesn't generate any styling at all 💡

Reply:
Yes and yes. I've added one test https://github.com/warp-ds/drive/pull/32/files#diff-1d3b2b3a27491b52437b1e12c9596a2a8f058641c4cf23c1d4455a8b14def09fR100 for the "out of bounds" test but we will add more of those. Makes total sense

Reply:
What do you think about this approach https://github.com/warp-ds/drive/pull/32/commits/c130ff22eb6635fea6831f90ce9d28172960609b @nwikstrand @pearofducks